WolvenKit.Common/Tools/ProcessUtil.cs

```diff
@@ -13,7 +13,7 @@ public static class ProcessUtil
     /// <param name="fileName"></param>
     /// <param name="arguments"></param>
     /// <returns>true if exit code is 0</returns>
-    public static async Task<bool> RunProcessAsync(string fileName, params string[] arguments)
+    public static async Task<bool> RunProcessAsync(string fileName, string arguments, string workingDir = "")
```

Comment:
```suggestion
    public static async Task<bool> RunProcessAsync(string fileName, string arguments, string workingDir = string.Empty)
```

Reply:
"Default parameter value needs to be compile-time constant"